website/docs/sdk-reference/elixir.md

```diff
@@ -405,13 +610,23 @@ values = ConfigCat.get_all_values(
 )
 ```
 
+## `get_all_value_details()`
```

Comment:
Should `get_value_details()` be listed here as well?

Reply:
We already mentioned that in the `Anatomy of get_value_details()` section.